semantic/src/site/globals/site.overrides

```diff
@@ -123,6 +123,38 @@ p strong {
   margin-left: 10px;
 }
 
+/* Application home */
+#application-area {
+  font-family: @userAreaFont;
```

Comment:
FYI -- @userAreaFont is not the OpenSans one, it's the one used in the Header area. So you probably don't want to set this for the whole `#application-area`. 

I'd say make a class called `.alt-font`, with just `font-family: @userAreaFont`, then apply it in conjuction with the appropriate `<h>` tags, e.g. for the company title, you'd have:
`<Header as="h2" className=".alt-font">{company.name...}`

Reply:
Done... althought instead of using the class .alt-font I added as one of the properties on `.company-title`